pkg/metrics/procs_windows.go

```diff
@@ -726,25 +801,44 @@ func (self *ProcsMonitor) Sample() (results sample.EventBatch, err error) {
 				}
 
 				// Process Status For Windows
-				sample.Status, err = self.getStatus(pid)
-				if err != nil {
-					logSampleError(pid, winProc, err, "can't get process status")
+				if wmiProcDataEnabled {
+					sample.Status = *winProc.Status
+				} else {
+					sample.Status, err = self.getStatus(pid)
+					if err != nil {
+						logSampleError(pid, winProc, err, "can't get process status")
+					}
 				}
 
-				currentProcessTime, err := self.getTimes(pid)
-				if err != nil {
-					logSampleError(pid, winProc, err, "can't get process times")
-				} else {
+				var currentProcessTime *SystemTimes
+				{
```

Comment:
out of curiosity. Is there any reason why this block exists? ☝️

Reply:
Would you like me to add a comment in the code explaining this behavior?

Reply:
ouch 🤦 , I didn't see the `goto` .... 🙈  I feel your pain ;-) 

>Would you like me to add a comment in the code explaining this behavior?

Yes please, a small sentence there would help and it's simple to add, thanks!

Reply:
Done!